spec/models/manageiq/providers/hawkular/middleware_manager_spec.rb

```diff
@@ -187,4 +187,104 @@ def timeline_domain_expectations(status)
       end
     end
   end
+
+  describe "server group ops" do
```

Comment:
See #9. You may need to adapt these ones once #9 is merged.

Reply:
Also it tests implementation details, like private methods. Should be avoided.